src/v2/Apps/Artist/Routes/Works/index.tsx

```diff
@@ -94,6 +94,8 @@ export const WorksRouteFragmentContainer = createFragmentContainer(WorksRoute, {
         sizes: { type: "[ArtworkSizes]" }
         sort: { type: "String", defaultValue: "-partner_updated_at" }
         width: { type: "String" }
+        locationCities: { type: "[String]" }
+        additionalGeneIDs: { type: "[String]" }
```

Comment:
leaving this out meant that navigating to the artist page, works tab, with a filter on this param provided in the URL _didn't_ actually include that in the request